docs-ui/stories/core/colors/tables.tsx

```diff
@@ -0,0 +1,257 @@
+import {useContext} from 'react';
+import styled from '@emotion/styled';
+import Sample, {SampleThemeContext} from 'docs-ui/components/sample';
+
+import space from 'app/styles/space';
+
+import ColorSwatch from './colorSwatch';
+
+type ColorDefinition = {
+  name: string;
+  lightValue: string;
+  darkValue: string;
+};
+
+type ColorGroup = {
+  id: string;
+  colors: ColorDefinition[];
+};
+
+const Wrap = styled('div')`
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(16em, 1fr));
+  grid-gap: ${space(2)};
+
+  & > * {
+    grid-column-end: span 1;
+  }
+`;
+
+const ColorTable = ({colorGroups}: {colorGroups: ColorGroup[]}) => {
+  const theme = useContext(SampleThemeContext);
+
+  return (
+    <Wrap>
+      {colorGroups.map(group => {
+        return <ColorSwatch key={group.id} colors={group.colors} theme={theme} />;
+      })}
+    </Wrap>
+  );
+};
+
+const neutralColors: ColorGroup[] = [
+  {
+    id: 'gray500',
+    colors: [
+      {
+        name: 'Gray 500',
+        lightValue: '#2B2233',
+        darkValue: '#EBE6EF',
+      },
+    ],
+  },
```

Comment:
Will you be updating these later to source them from theme?

Reply:
Yepp! Will update these once #29917 is approved and merged.